seqr/views/utils/orm_to_json_utils.py

```diff
@@ -5,11 +5,25 @@
 import json
 import logging
 import os
+from django.db.models.fields.files import ImageFieldFile
 
+from seqr.models import CAN_EDIT
+from seqr.views.utils.json_utils import _to_camel_case
 from family_info_utils import retrieve_family_analysed_by
 logger = logging.getLogger(__name__)
 
 
+def _get_json_for_record(record, fields, processed_fields={}, get_record_field=None, get_parent_guid=None):
```

Comment:
Not sure I understand the reason for the processed_fields, get_record_field, get_parent_guid args or why the added complexity is justified for these getters. Would be nice to whiteboard it.

Reply:
I'm not convinced the added complexity is justified, tbh. The idea is that the same function would be used for getting all the record fields from an object and from a database cursor because there were a lot of bugs coming up because they weren't quite identical